indexer/scripts/elastic-conf.py

```diff
@@ -0,0 +1,106 @@
+"""
+configure Elasticsearch
+"""
+
+import argparse
+import json
+import subprocess
+import time
+from collections import Counter
+from logging import getLogger
+from typing import Any, Dict, List, cast
+
+from elastic_transport import ConnectionError, ConnectionTimeout
+from elasticsearch import Elasticsearch
+
+from indexer.app import App
+from indexer.elastic import ElasticMixin
+
+logger = getLogger("elastic-stats")
+
+
+class ElasticConf(ElasticMixin, App):
+    def main_loop(self) -> None:
+        es = self.elasticsearch_client()
+        assert es.ping(), "Failed to connect to Elasticsearch"
+
+        index_template_path = "./conf/elasticsearch/create_index_template.json"
+        ilm_policy_path = "./conf/elasticsearch/create_ilm_policy.json"
+        initial_index_template = "./conf/elasticsearch/create_initial_index.json"
```

Comment:
Isn't it better to have conf env var that can be reused e.g. ELASTICSEARCH_CONF_DIR, then:
```suggestion
        index_template_path = f"{ELASTICSEARCH_CONF_DIR}/create_index_template.json"
        ilm_policy_path = f"{ELASTICSEARCH_CONF_DIR}/create_ilm_policy.json"
        initial_index_template = f"{ELASTICSEARCH_CONF_DIR}/create_initial_index.json"
```

Or something like that.